addons/account/tests/test_account_payment_register.py

```diff
@@ -115,7 +150,14 @@ def setUpClass(cls, chart_template_ref=None):
             'partner_id': cls.partner_a.id,
             'invoice_line_ids': [(0, 0, {'product_id': cls.product_a.id, 'price_unit': 1600.0, 'tax_ids': []})],
         })
-        cls.in_refund_1.action_post()
+        cls.in_refund_2 = cls.env['account.move'].create({
+            'move_type': 'in_refund',
+            'date': fields.Date.today(),
+            'invoice_date': fields.Date.today(),
+            'partner_id': cls.partner_a.id,
```

Comment:
You lack the same currency as the in_invoice associated to be in the same batch. 
Either you add it here, or you remove it from the bills you created. Probably best to remove it from the bills above, as we don't need foreign currency

Reply:
Yep that would make sense, I will remove from the bills and push back up. Should pass after that.